code/packaging/app/pom.xml

```diff
@@ -8,4 +8,11 @@
         <relativePath>../app-parent/pom.xml</relativePath>
     </parent>
     <packaging>war</packaging>
+
+    <dependencies>
```

Comment:
As mentioned before we don't use the MySQL driver but instead the MariaDB one.  Besides that why did you need to add a dependency here?